avocado/core/runner.py

```diff
@@ -440,16 +440,17 @@ def sigtstp_handler(signum, frame):     # pylint: disable=W0613
                 ctrl_c_count += 1
                 if ctrl_c_count == 1:
                     if not stage_1_msg_displayed:
-                        abort_reason = "Interrupted by ctrl+c"
+                        abort_reason = "Interrupted by signal/SIGINT"
```

Comment:
Well to me this looks unnecessary too complex. Those who send signals know it's an equivalent and those who just wanted to interrupt the whole thing might be confused.